openlibrary/templates/lists/widget.html

```diff
@@ -152,32 +152,32 @@
             <div class="dropdown">
               <div class="read-statuses">
                 $if users_work_read_status == 2 or users_work_read_status == 3:
-                  <form method="POST" action="$(work_key)/bookshelves.json?debug=true">
+                  <form class="readingLog" method="POST" action="$(work_key)/bookshelves.json?debug=true">
                     <input type="hidden" name="action" value="add"/>
                     <input type="hidden" name="redir" value="true"/>
                     <input type="hidden" name="bookshelf_id" value="1"/>
                     $if edition_key:
-                      <input type="hidden" name="edition_id" value="$(edition_key)"/>
+                      <input type="hidden" id="edition_id" name="edition_id" value="$(edition_key)"/>
                     <button class="nostyle-btn" type="submit">$_('Want to Read')</button>
                   </form>
 
                 $if users_work_read_status != 2:
-                  <form method="POST" action="$(work_key)/bookshelves.json?debug=true">
+                  <form class="readingLog" method="POST" action="$(work_key)/bookshelves.json?debug=true">
                     <input type="hidden" name="action" value="add"/>
                     <input type="hidden" name="redir" value="true"/>
                     <input type="hidden" name="bookshelf_id" value="2"/>
                     $if edition_key:
-                      <input type="hidden" name="edition_id" value="$(edition_key)"/>
+                      <input type="hidden" id="edition_id" name="edition_id" value="$(edition_key)"/>
```

Comment:
Why do these have an ID now?

Reply:
@tabshaikh ? I'm assuming it's not required. I don't see any js using it in `git grep`
```suggestion
                      <input type="hidden" name="edition_id" value="$(edition_key)"/>
```